registry/storage/driver/s3-aws/s3.go

```diff
@@ -148,6 +149,11 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		return nil, fmt.Errorf("Invalid region provided: %v", region)
 	}
 
+	regionEndpoint, ok := parameters["regionendpoint"]
```

Comment:
Should only one of `region` and `regionendpoint` be provided? If so, you should change the check for the `region` parameter above as well as the documentation to reflect this.